index.html

```diff
@@ -1,289 +1,647 @@
+
+            </div>
+
+        </div>
+
+        <div class="card-img-container">
+            <img src="cardTwoImage.svg" alt="" class="card-img" />
+        </div>
+
+
+
+      </div>
+
+
+      <!-- single card ends here -->
+
+    </section>
+    <!-- card Two Session ends here -->
+
+  
+
+    <!-- About session starts -->
+    <div class="bottmo-session">
+    <section id="about" class="about">
+      <div class=" sub-section">
+        <h2 class="intro">
+          About
+          <br />
+          Myself
+        </h2>
+
+        <p class="about-me-details">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <h4 class="social-media-intro">LET’S CONNECT</h4>
+
+        <ul class="social-media-icon">
+          <li class="social-media">
+            <a class="list-style" href="#"><img src="tweeter.svg" alt="" /></a>
+          </li>
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="Linkedin-icon.svg" alt=""
+            /></a>
+          </li>
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="mediaThree.svg" alt=""
+            /></a>
+          </li>
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="mediaFour.svg" alt=""
+            /></a>
+          </li>
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="mediaFive.svg" alt=""
+            /></a>
+          </li>
+        </ul>
+
+        <button type="submit" class="btnNormal btnNormalTwo">
+          Get my resume
+        </button>
+      </div>
+      <div class="languages">
+        <ul class="lang-sec">
+          <li class="lang-tille-link">
+            <h4 class="lang-title">Languages</h4>
+
+            <img
+              class="language-display-icon"
+              src="lanDisplayIcon.svg"
+              alt=""
+            />
+          </li>
+        </ul>
+
+        <ul class="lang-icon">
+          <li class="langIconDetails first-lang">
+            <img class="iconDetailsImg" src="javascriptImage.svg" alt="" />
+            <a class="langIconDetailsLink" href="#"
+              ><h4 class="lang-icon-">JavaScript</h4></a
+            >
+          </li>
+          <li class="langIconDetails sec-lang">
+            <img class="iconDetailsImg" src="htmlImage.svg" alt="" />
+            <a class="langIconDetailsLink" href="#"
+              ><h4 class="lang-icon-">HTML</h4></a
+            >
+          </li>
+          <li class="langIconDetails thd-lang">
+            <img class="iconDetailsImg" src="cssImage.svg" alt="" />
+            <a class="langIconDetailsLink" href="#"
```

Comment:
- [ ] Please kindly ensure that the language(HTML, CSS, JS)  icons are the same size as the original design. At the moment, yours is looking too big.

### Your Design 
![image](https://user-images.githubusercontent.com/43995830/165737604-ce88543d-fb1a-49cd-b2b5-1d5e3679570d.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/165738225-967240e3-184d-4c4a-b8e8-40e3bc174d5f.png)